src/main/java/org/jabref/logic/importer/fileformat/PdfContentImporter.java

```diff
@@ -372,11 +375,13 @@ Optional<BibEntry> getEntryFromPDFContent(String firstpageContents, String lineS
         String volume = null;
         String number = null;
         String pages = null;
+        String arXivId = null;
         // year is a class variable as the method extractYear() uses it;
         String publisher = null;
 
         EntryType type = StandardEntryType.InProceedings;
         if (curString.length() > 4) {
+            arXivId = getArXivId(null);
```

Comment:
Passing null is really bad style. This neesd to be fixed eventually!!

I will approve nevertheless. Too much time spend on this.